app/src/androidTest/java/com/github/orkest/PersistenceTesting.kt

```diff
@@ -114,4 +130,16 @@ class MockSongDao: AppDao.Companion.SongDao {
     override fun clear() {
         songList.clear()
     }
+}
+
+class MockPostDao: AppDao.Companion.PostDao{
+    private val postList = mutableListOf<AppEntities.Companion.PostEntity>()
+
+    override fun getAllPosts(): List<AppEntities.Companion.PostEntity> {
+        return postList
+    }
+
+    override fun insertPosts(posts: List<AppEntities.Companion.PostEntity>) {
+        postList.addAll(posts)
+    }
 }
```

Comment:
Very good tests, though do you think that considering more edge cases like null values or empty lists would have been interesting ?